en/flight_modes/land.md

```diff
@@ -32,24 +32,110 @@ Parameter | Description
+
+### Flaring / roll out
+
+Flaring consists of a switch from altitude tracking to a shallow sink rate setpoint and constraints on the commandable throttle, resulting in nose up manuevering to slow the descent and produce a softer touchdown.
+
+The flaring altitude is calculated during the final approach via "time-to-impact" ([FW_LND_FL_TIME](../advanced_config/parameter_reference.md#FW_LND_FL_TIME)) and the approach descent rate.
+An additional safety parameter [FW_LND_FLALT](../advanced_config/parameter_reference.md#FW_LND_FLALT) sets the minimum altitude at which the vehicle will flare (if the time based altitude is too low to allow a safe flare maneuver).
+
+If belly landing, the vehicle will continue in the flaring state until touchdown, land detection, and subsequent disarm. For runway landings, [FW_LND_TD_TIME](../advanced_config/parameter_reference.md#FW_LND_TD_TIME) enables setting the time post flare start to pitch down the nose (e.g. consider tricycle gear) onto the runway ([RWTO_PSP](../advanced_config/parameter_reference.md#RWTO_PSP)) and avoid bouncing. This time roughly corresponds to the touchdown post flare, and should be tuned for a given airframe during test flights only after the flare has been tuned.
+
+Parameter | Description
+--- | ---
+[FW_LND_FL_TIME](../advanced_config/parameter_reference.md#FW_LND_FL_TIME) | Time before impact (at current descent rate) at which the vehicle should flare. 
+[FW_LND_FL_SINK](../advanced_config/parameter_reference.md#FW_LND_FL_SINK) | A shallow sink rate the aircraft will track during the flare.
+[FW_LND_FLALT](../advanced_config/parameter_reference.md#FW_LND_FLALT) | Minimum altitude above ground the aircraft must flare. This is only used when the time-based flare altitude is too low.
+[FW_LND_FL_PMAX](../advanced_config/parameter_reference.md#FW_LND_FL_PMAX) | Maximum allowed pitch during the flare.
+[FW_LND_FL_PMIN](../advanced_config/parameter_reference.md#FW_LND_FL_PMIN) | Minimum allowed pitch during the flare (often necessary to avoid negative pitch angles commanded to increase airspeed, as the throttle is reduced to idle setting.)
+[FW_LND_TD_TIME](../advanced_config/parameter_reference.md#FW_LND_TD_TIME) | The time after flare start when the vehicle should pitch the nose down.
+[RWTO_PSP](../advanced_config/parameter_reference.md#RWTO_PSP) | Pitch setpoint while on the runway. For tricycle gear, typically near zero. For tail draggers, positive.
+[FW_THR_IDLE](../advanced_config/parameter_reference.md#FW_THR_IDLE) | Idle throttle setting. The vehicle will retain this setting throughout the flare and roll out.
+
+### Abort
+
+#### Operator abort
+
+The landing may be aborted by the operator at any point during the final approach, resulting in a climb out to an orbit pattern centered above the land waypoint.

```

Comment:
Should we mention here how the operator abort can be triggered? (by pressing the "Abort landing" button on QGC, which appears when currently in a landing sequence). It's though ofc groundstation specific, thus not that sure if it belongs here.

Reply:
https://github.com/PX4/PX4-user_guide/pull/2245/commits/3f4a09ab88b7ce1cc71a736130ccbb442a2f2dc1